site/docs/components/segmented-button-group/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Segmented Button Group
+data:
+  description: "`SegmentedButtonGroup` is a component that displays a list of actionable buttons, flush with separators."
+
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/blob/main/packages/lab/src/segmented-button-group/SegmentedButtonGroup.tsx"
+  package:
+    name: "@salt-ds/lab"
+  alsoKnownAs: ["Action", "UI action"]
```

Comment:
is segmented button the replacement of uitk split button? if so, it might be good to add it here too

Reply:
Split button will be a pattern based on Segmented Button Group - but yes makes sense to include it here